src/App.tsx

```diff
@@ -1,7 +1,37 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  key: string;
+};
+
+export class App extends React.Component<{}, State> {
+  state: Readonly<State> = {
+    key: '',
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', this.handleKeyup);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', this.handleKeyup);
+  }
+
+  handleKeyup = (event:KeyboardEvent) => {
+    this.setState({ key: event.key });
+  };
+
+  render() {
+    const { key } = this.state;
+
+    return (
+      <div className="App">
+        <p className="App__message">
+          {key === ''
+            ? 'Nothing was pressed yet'
+            : `The last pressed key is [${key}]`}
```

Comment:
I guess the better way is to manage this text depending directly from on the state value, not to make additional statement.